Taskfile.yml

```diff
@@ -2,8 +2,8 @@ version: '3'
 
 vars:
   ALPVER: 3.16
-  GOVER: 1.18
-  KAFKAVER: 1.6.2
+  GOVER: 1.19
+  KAFKAVER: 1.9.2
```

Comment:
The Kafka version is quite a bump right? Will that go well?

Reply:
I have no idea, but we don't use those images anymore, anyway.

My personal preference is usually to just try to roll the last version and revert if it doesn't work, vs stick to one and hope that there are no vulnerabilities.

I don't mind keeping at 1.6.2. Whatever you like! Let me know.

Reply:
Let's upgrade indeed.